src/Time/Time.js

```diff
@@ -104,86 +116,96 @@ class Time extends Component {
                 id={id}>
                 {/* Hours */}
                 {showHour ? (
-                    <TimeItem
-                        defaultValue={1}
-                        disabled={disabled}
-                        disableStyles={disableStyles}
-                        downButtonProps={hoursDownButtonProps}
-                        format12Hours={format12Hours}
-                        inputProps={hoursInputProps}
-                        localizedText={localizedText}
-                        max={max}
-                        name='hour'
-                        placeholder={'hh'}
-                        spinners={spinners}
-                        time={time}
-                        type={'Hours'}
-                        upButtonProps={hoursUpButtonProps}
-                        updateTime={this.updateTime}
-                        value={time.hour} />
+                    <div className='fd-time__col' onClick={() => this.updateActiveColumn('hour')}>
+                        <label className='fd-time__slider-label'>Hours</label>
+                        <TimeItem
+                            active={active}
+                            defaultValue={1}
+                            disabled={disabled}
+                            disableStyles={disableStyles}
+                            downButtonProps={hoursDownButtonProps}
+                            format12Hours={format12Hours}
+                            inputProps={hoursInputProps}
+                            localizedText={localizedText}
+                            max={max}
+                            name='hour'
+                            time={time}
+                            type={'Hours'}
```

Comment:
Could we combine name and type and just have one prop or is there another purpose for them? I know those pre-existed this pr so we can totally make it a followup.